Cubical/Relation/Binary/Order/Poset/Instances/PosetalReflection.agda

```diff
@@ -0,0 +1,109 @@
+{-# OPTIONS --cubical --safe #-}
+
```

Comment:
Can you add a brief explanation what a posetal reflection is (in particular, what you reflect in)?